Dockerfile

```diff
@@ -1,5 +1,9 @@
-FROM ubuntu:16.04
-ENV ELIXIR_VERSION=1.10.3-otp-22
+FROM ubuntu:20.04
+ENV ELIXIR_VERSION=1.11.3-otp-23
+
+# Set Timezone so TZ Info doesn't hault the build
+ENV DEBIAN_FRONTEND="noninteractive"
+ENV TZ="America/Chicago"
```

Comment:
:japanese_goblin: any reason not to use UTC?

Reply:
Not really - picked a zone close to us. UTC might be a better option.

Reply:
That goblin is amazing.

Reply:
Looks like it builds UTC by default, so just rolling with that.